clulab/eidos
# Temporal attachment

Failing tests:

```
- TestDoc3
-- Paragraph 14
--- "should have correct edges 4"
--- "should have correct edges 5"
```

To reproduce the error:

1. Copy "timenorm_model.hdf5" file into src/main/resources/org/clulab/wm/eidos/models/
2. Change "B-Time" to "B-Timex" in line 42 of EidosDocument.scala
3. Change "B-Time" to "B-Timex" in line 20 of TemporalAttachement.yml

The tests pass with one of the following changes in the text of paragraph 14:

- Ta'izz -> Taizz
- and/or -> and
